### DISCUSSION TITLE
New extension for SAP HANA

### DISCUSSION THREAD

**Discussion by akirafujiu** [slick/slick#2669] [Questions] [answered]
I'm interested in using SAP HANA via Slick, but currently there is no official support from Slick as you know. I found this repo https://github.com/sbcd90/slick-hana which has not been maintained recently. If I try to add this with some efforts to Slick itself as another profile like Oracle, what are necessary steps or missing pieces in this repository?

Actually SAP HANA can be connected via JDBC as https://help.sap.com/docs/SAP_HANA_PLATFORM/0eec0d68141541d1b07893a39944924e/ff15928cf5594d78b841fbbe649f04b4.html, just guessing that we can enhance Slick to handle SAP HANA properly.

I'm just an user of Slick, and not sure if I can implement well whole pieces to bring this repo to Slick, but it would be very helpful if someone give some advices.  

Thanks in advance!

**Comment by nafg** [comment]
What is the advantage over it being a separate repo?

**Reply by akirafujiu** [reply]
SAP HANA is one of the major database in the market of relational database, then I thought it better to be worth supporting fully.

**Reply by nafg** [reply]
Supported fully by who?

**Reply by akirafujiu** [reply]
I'm going to work at least, but I'm not sure if I can cover everything, since I'm just a user of Slick as wrote. That's why I asked this. 

>  what are necessary steps or missing pieces in this repository?

**Reply by nafg** [reply]
But if you can't then who should?

**Comment by nafg** [comment]
Do you want to discuss it on Discord?

**Reply by akirafujiu** [reply]
I'm fine with whichever, is it your preference?

**Reply by nafg** [reply]
If we can VC there it might be helpful

**Comment by akirafujiu** [comment, marked as answer]
I'm fine to close this
